pkgs/applications/networking/instant-messengers/pidgin-plugins/purple-signald/default.nix

```
@@ -0,0 +1,32 @@
+{ lib, stdenv, fetchgit, fetchFromGitHub, pidgin, glib, json-glib, signald }:
```

Looks like this is unused.
```suggestion
{ lib, stdenv, fetchgit, fetchFromGitHub, pidgin, json-glib, signald }:
```